crates/sui-indexer/src/store/indexer_store.rs

```diff
@@ -48,6 +48,12 @@ pub trait IndexerStore: Clone + Sync + Send + 'static {
         object_changes: Vec<TransactionObjectChangesToCommit>,
     ) -> Result<(), IndexerError>;
 
+    async fn persist_full_objects_history(
+        &self,
+        object_changes: Vec<TransactionObjectChangesToCommit>,
+    ) -> Result<(), IndexerError>;
+
+    // persist objects snapshot with object changes during backfill
```

Comment:
we don't really have a backfill mode anymore right? instead, objects snapshot processor uses this command to update the snapshot table as needed